src/julia-syntax.scm

```diff
@@ -810,14 +810,16 @@
                   (sig    (car temp))
                   (params (cdr temp)))
              (if (pair? params)
-                 (let* ((lnos (filter (lambda (e) (and (pair? e) (eq? (car e) 'line)))
-                                      body))
-                        (lno (if (null? lnos) '() (car lnos))))
-                   (syntax-deprecation #f
-                                       (string "inner constructor " name "(...)" (linenode-string lno))
-                                       (deparse `(where (call (curly ,name ,@params) ...) ,@params)))))
+                 (syntax-deprecation #f
```

Comment:
either here on in deprecated.jl, should these be annotated with "remove in version ..." ?